lib/views/docs.jade

```diff
@@ -56,7 +56,10 @@ mixin props(properties)
                     dl.meta
                         each key in meta
                             dt= key
-                            dd= details[key]
+                            if key === 'errorCodes'
+                                dd= JSON.stringify(details[key], null, 2).substr(3, JSON.stringify(details[key], null, 2).length -5)
```

Comment:
We're stringifying twice. How about iterating over the values and creating a new `dd` for each?